tests/integration_tests/process_input_tests.sh

```diff
@@ -86,6 +87,47 @@ test_input_directories_with_wildcard() {
     assertContains "Actual messages:$actual_message Did not contain the expected message.\n" "$actual_message" "$expected_message2"
 }
 
+test_severity_invalid_mode(){
+    input_paths="./test_data/test_dir_1/valid_script_error.sh"
+    severity_mode="verbose"
+    local expected_message1="Error setting unknown severity mode. Defaulting severity mode to style."
+    local expected_message2="ERROR: ShellCheck detected issues"
+    local actual_message=$(process_input)
+
+    assertContains "Did not find the message." "$actual_message" "$expected_message1"
+    assertContains "Did not find the message." "$actual_message" "$expected_message2"
+}
+
+test_error_file_severity_error(){
```

Comment:
Regarding the last three tests, I think a true test would be to have a single file with 4 issues with different severity level; one error, one warning, one info, and one style issue. Then set the severity_mode and check if the correct issues show up.

Also, the name of the test should be descriptive of what is being tested, if you use the single file approach, you can then name the tests like test_severity_mode_warning, test_error_severity_mode, etc

Reply:
Updated the tests, added additional tests and changed the test name